pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -725,9 +749,10 @@ func conditionallyConvertToIstioMtls(
 		if len(subjectAltNamesToUse) == 0 {
 			subjectAltNamesToUse = serviceAccounts
 		}
-		return buildIstioMutualTLS(subjectAltNamesToUse, sniToUse, proxy)
+		// TODO GregHanson enableTransportSocketMtls always true mode=Istio_Mutual case?
```

Comment:
is this todo still unresolved?

Reply:
yes, still a TODO.  until I can test the changes with the updated envoy binary - I can't verify one way or the other